packages/fxa-auth-server/test/local/server.js

```diff
@@ -333,9 +325,9 @@ describe('lib/server', () => {
             });
 
             it('second request has its own locale', () => {
-              assert.equal(translator.getLocale.callCount, 0);
-              assert.equal(secondRequest.app.locale, 'fr');
-              assert.equal(translator.getLocale.callCount, 1);
+              // Note that fr-CH would be the correct language, but it is not in the list of supported
+              // languages. This means that en-GB has the highest q-value and should be selected.
```

Comment:
Thanks for the comments 👍 FWIW we made a note in that "Modernize Localization" epic / that spike we've both commented on around reviewing that list (and, Peter filed a separate issue for it, in that epic now).